### PR TITLE
hide segmented reports are pre-processed

### LINKED ISSUE #15036: Hide "AND segmented reports are pre-processed (faster, requires cron)" from the UI when enable_create_realtime_segments = 0
Currently the UI looks like this:
Screenshot from 2019-10-22 17-32-10

The last selector cannot be changed when `enable_create_realtime_segments = 0`

Because it provide no value in itself, it would be great to hide the text all together: "AND segmented reports are pre-processed (faster, requires cron)" from the UI when `enable_create_realtime_segments = 0`

### PR DESCRIPTION
**Pull request by geekdenz**
when enable_create_realtime_segments = 0
fixes #15036

### Description:

Please include a description of this change and which issue it fixes. If no issue exists yet please include context and what problem it solves.

### Review

* [ ] Functional review done
* [ ] Potential edge cases thought about (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] Usability review done (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] Security review done [see checklist](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] Code review done
* [ ] Tests were added if useful/possible
* [ ] Reviewed for breaking changes
* [ ] Developer changelog updated if needed
* [ ] Documentation added if needed
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/SegmentEditor/SegmentEditor.php [hunk 1]
```diff
@@ -316,6 +316,16 @@ public static function isAddingSegmentsForAllWebsitesEnabled()
...
 
+    /**
+     * Returns whether create realtime segments is enabled or not.
+     *
+     * @return bool
+     */
+    public static function isCreateRealtimeSegmentsEnabled(): bool
+    {
+        return Config::getInstance()->General['enable_create_realtime_segments'] == 1;
+    }
```
Comment (sgiehl) [inline plugins/SegmentEditor/SegmentEditor.php:327]: You could change this to use this new method as well:
https://github.com/matomo-org/matomo/blob/da44e3b3416f9012c774aa7abef19cb8cdbf58e1/plugins/SegmentEditor/API.php#L120-L121

Comment (geekdenz) [reply plugins/SegmentEditor/SegmentEditor.php:327]: Thanks @sgiehl .

Sorry but I don't know what you mean. The lines you sent are not a method. Do you mean not checking for `== 1`? I thought for BC that is what we wanted.

Comment (tsteur) [reply plugins/SegmentEditor/SegmentEditor.php:327]: @geekdenz what @sgiehl is meant replacing that logic to use the new method like 

```
$realTimeSegmentsEnabled = SegmentEditor:: isCreateRealtimeSegmentsEnabled();
```

Not the other way around.

**Review by sgiehl — APPROVED** [review]
Code looks fine and works as expected. We could maybe add another UI test that checks the UI when the option is disabled to ensure it doesn't regress in the future

**Comment by justinvelluppillai** [comment]
@geekdenz This PR is linked to #15036 and #15322, is that correct?

**Review by sgiehl — APPROVED** [review]

**Pull request merged**